test/integration/freetext_editor_spec.mjs

```diff
@@ -2354,14 +2354,14 @@ describe("FreeText Editor", () => {
             `${getEditorSelector(1)} .overlay.enabled`
           );
 
-          rect = await page.$eval(getEditorSelector(0), el => {
+          rect = await page.$eval(getEditorSelector(1), el => {
```

Comment:
Could you explain why the changes in this file are necessary and how they fix the issue? Looking at this test earlier I was a bit confused as to what this part of the test is achieving. Perhaps it could be resolved with more accompanying comments in the test to explain the intent of each "block" of logic (since the test is quite lengthy).

Reply:
When taking the rect for editor 0 which now contains "A" and then clicking at rect.x+5*rect.width,rect.y+5*rect.height doesn't guarantee to be on the second editor.
I probably copy/paste some code without thinking about this.